package.json

```diff
@@ -57,6 +57,7 @@
     "release:publish:dry-run": "pnpm publish --recursive --tag latest --registry=\"http://localhost:4873/\"",
     "release:tag": "node scripts/releaseTag.mjs",
     "release:prepare": "node scripts/createReleasePR.mjs",
+    "generate:exports": "node scripts/generateDeepExports.mjs",
```

Comment:
Why not call this script from the `build` command directly? This will avoid requiring users to run the script before committing.

Reply:
IMO this is better. It's easier to gain more control over it when it's part of source code, and it makes the build less magic.